docker/Dockerfile

```diff
@@ -0,0 +1,65 @@
+#Starting with the lastest stable geospatial R image as of 1/20/2022
+#Dockerfile references are here:
+#https://github.com/rocker-org/rocker-versioned2/tree/master/dockerfiles
+#includes Ubuntu 20.04 LTS (stable release)
+#includes R version 4.1.2
+#includes all packages and Linux libraries in the Dockerfile for verse, 
+#which is based on tidyverse, which is based on rstudio, which is based on r-ver.
+FROM rocker/geospatial:4.1.2
+
```

Comment:
It may be worthwhile to add the public key for the DOI's SSL Intercept. This helps the system work while users are connected to VPN. Cf. [here](https://code.usgs.gov/wma/wp/reanalysis-gefs/-/blob/main/docker/Dockerfile#L8-9) for an example.

Reply:
Thanks for the review! I'll add these changes